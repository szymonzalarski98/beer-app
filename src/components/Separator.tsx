import React from 'react';
import { View, StyleSheet } from 'react-native';
import { deviceWidth, colors } from '../styles/dimensions.js';

const Separator = () => <View style={styles.separator} />;

export default Separator;

const styles = StyleSheet.create({
  separator: {
    marginHorizontal: deviceWidth * 0.05,
    height: 1,
    backgroundColor: colors.separator,
  },
});