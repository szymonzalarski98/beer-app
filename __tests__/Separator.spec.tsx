import React from 'react';
import { render } from 'react-native-testing-library';
import Separator from '../src/components/Separator';

describe('Test Separator component', () => {
  it ('Should renders correctly', () => {
    const { getByTestId, baseElement } = render(<Separator />);
    expect(getByTestId('separator')).not.toBeNull();
    expect(baseElement).toMatchSnapshot();
  });
});