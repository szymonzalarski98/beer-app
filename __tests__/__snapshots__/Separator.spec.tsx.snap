// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Separator component Should renders correctly 1`] = `undefined`;
